src/main/java/api/WelcomeController.java

```diff
@@ -0,0 +1,27 @@
+package api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api/v1/library/welcome")
+public class WelcomeController {
+
+    @GetMapping("")
+    public ResponseEntity<Map<String, String>> welcome() {
+        Map<String, String> response = new LinkedHashMap<>();
+        response.put("message", "Welcome to the library");
+        response.put("date", new SimpleDateFormat("dd.MM.yyyy").format(Calendar.getInstance().getTime()));
+        return new ResponseEntity<>(response, HttpStatus.OK);
+
+    }
```

Comment:
пожалуйста обращай внимание на версию языка, на котором ты пишешь код!
У нас java 17 - используй все новые фичи!
И обрати внимание на внутренности класса `ResponseEntity` - у него там много удобных методов.
```suggestion

  public ResponseEntity<Map<String, String>> welcome() {
        var response = Map.of(
            "message", "Welcome to the library",
            "date", new SimpleDateFormat("dd.MM.yyyy").format(Calendar.getInstance().getTime())
        );
        return ResponseEntity.ok(response);
    }
```

В дальнейшем я буду просто писать что-то вроде "перделай на java 17" - не буду сходу писать готовый код.

Reply:
Постоянно забываю о var, буду стараться не допускать таких ошибок